VotingApplication/VotingApplication.Web/Scripts/Controllers/BasicVoteController.js

```diff
@@ -53,6 +61,15 @@
 
         }
 
+        function addOption() {
```

Comment:
We might want to factor this out to the parent Vote controller, as it seems to be identical in each class